### PR TITLE
Feature/queries

### PR DESCRIPTION
**Pull request by thasipablo**
**Tasks:**

- Created a database named `vet_clinic`
- Created a table named `animals`
- Created `.sql` files, `shema.sql`, `queries.sql`, and `data.sql` with related queries

**Screen shots of the execution of queries:**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+  id INT, 
+  name VARCHAR,
```
Comment (Shedrack-Sunday) [inline schema.sql:5]: - [ ] Setting limits on `VARCHAR` columns in a database ensures data integrity, improves performance, and enhances security by preventing overly long or inappropriate data. Kindly add a limit to the name field 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @thasipablo           👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the SQL  features ✔️ 
- Nice use query parameters✔️ 

# Required Changes  🌵  

- [x] You've done well, However, kindly follow the suggestion on the right query pattern properly so your impressive work will have a better implementation 👍 


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (thasipablo) [reply schema.sql]: Thank you @Shedrack-Sunday .
The update is done.

**Review by TadesseDev — APPROVED** [review]
### Hi @thasipablo 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**